### ISSUE TITLE
QStandardPaths: XDG_RUNTIME_DIR not set, defaulting to '/tmp/runtime-root'

### ISSUE THREAD

**Issue by NaturalHate** [benjamimgois/goverlay#134]
I use Manjaro and the AUR package. Those issues popped up since the newer versions that changed how the global enable works.
First one being that the program does not start, I have to open a terminal and sudo goverlay it, which throws me the error in the title. Without sudo I get an access violation error:
```
[FORMS.PP] ExceptionOccurred 
  Sender=EAccessViolation
  Exception=Access violation
  Stack trace:
  $0000000000576F7B
[FORMS.PP] ExceptionOccurred 
```

The MangoHud settings within Goverlay stay after restarting the program, and work within the testing applications, however they do not affect the actual overlay within the games, which still run the settings from the older versions.
The fix for the power draw on Intel cpus also is not persistent and has to be toggled after every reboot, but that isn't a new issue, so I don't think it is related to the error.

**Comment by benjamimgois** [comment]
That's a strange behavior, i also use Manjaro and do not have those problems. First, try to whipe all mangohud and goverlay configs:

```
rm -r $HOME/.config/goverlay
rm -r $HOME/.config/MangoHud
rm -r $HOME/.bash_profile_bkp
rm -r $HOME/.profile_bkp
rm -r /tmp/goverlay
```
Now remove the line `export MANGOHUD=1` from .profile and .bash_profile inside your home folder.

Restart you machine. Now you're back into a blanck config of mangohud and goverlay. Open goverlay and make a new config, see if the problems persists.

**Comment by NaturalHate** [comment]
```
rm: cannot remove '/home/*****/.bash_profile_bkp': No such file or directory
rm: cannot remove '/home/*****/.profile_bkp': No such file or directory
rm: cannot remove '/tmp/goverlay': No such file or directory
```
There was a .profile.bkp though? Not sure they do anything, assuming bkp means backup.
The sudo & path error issue persists. Goverlay still has the same MangoHud settings enabled from before, but now it does not work in any game anymore, not even the previous config that was showing before.
Tried toggling the global enable off and on, restart, same issue.
Tried toggling the global enable off, restart, same issue and in Goverlay it still showed it as enabled (I clicked "save" every time).
There's neither a goverlay nor MangoHud folder within ,config anymore.

**Comment by benjamimgois** [comment]
@NaturalHate , you shouldn't use `sudo ` or root to run goverlay. Using it, the configs might being created inside the `/root/.config/`, that's why the folders aren't appearing. Are you on **Wayland** ? if you are, that is the reason of that error "...Exception ocurred...". There's a observation about Wayland on the main page. Try execute goverlay with the command:

`QT_QPA_PLATFORM=xcb goverlay --style fusion`

It should fix all the issues.

**Comment by NaturalHate** [comment]
Okay, I think it is fixed now. I did however had to do another restart, as before that I got the following behavior and error messages, which I wrote down before the restart. Not sure if it is relevant for further bug fixing but I did not want to cut it out, just in case:


I do indeed use Wayland, yes.
When I enter the command I get this [error](https://i.imgur.com/ubhumzr.png).
If I ignore it / click ok it starts Goverlay, but when I save I get this [error](https://i.imgur.com/O1LsuEq.png).
The test programs do not get any of the applied settings after this, and restarting Goverlay using the command will also reset it to a blank slate.
It created a goverlay folder in .config, but it only contains a reshade_git text file within the initial_values folder.
It also created a MangoHud folder in .config, but the MangoHud.conf also only contains the basic values.
